static/js/compose.js

```diff
@@ -11,6 +11,8 @@ const render_compose_private_stream_alert = require("../templates/compose_privat
 const rendered_markdown = require("./rendered_markdown");
 const util = require("./util");
 
+const MAX_MESSAGE_LENGTH = 10000;
```

Comment:
This reminds me -- we should probably include this value in `page_params/register` under this name so that it's available to mobile/terminal clients?  I think 50% of the work will be the API docs :), @orientor may be able to help with that.